app/assets/stylesheets/sail/settings.scss

```diff
@@ -126,17 +127,19 @@
     }
 
     .btn-value-submit {
-      width: 100%;
+      width: 95%;
```

Comment:
Making the buttons smaller makes them look a little awkward in respect to the input fields (which occupy 100%).

I would say we can either decrease the inputs to 95% as well, so that they are properly aligned and we have more free space in the cards, or go back to 100%.